server/src/test/java/org/apache/iotdb/db/engine/storagegroup/TsFileProcessorTest.java

```diff
@@ -322,6 +324,37 @@ public void alignedTvListRamCostTest()
     Assert.assertEquals(721560, memTable.memSize());
   }
 
+  @Test
+  public void testMemTableMapRamCost()
+      throws MetadataException, WriteProcessException, IOException {
+    processor =
+        new TsFileProcessor(
+            storageGroup,
+            SystemFileFactory.INSTANCE.getFile(filePath),
+            sgInfo,
+            this::closeTsFileProcessor,
+            (tsFileProcessor) -> true,
+            true);
+    TsFileProcessorInfo tsFileProcessorInfo = new TsFileProcessorInfo(sgInfo);
+    processor.setTsFileProcessorInfo(tsFileProcessorInfo);
+    this.sgInfo.initTsFileProcessorInfo(processor);
+    SystemInfo.getInstance().reportStorageGroupStatus(sgInfo, processor);
+    processor.insertTablet(genInsertTablePlan(0, false), 0, 10, new TSStatus[10]);
+    IMemTable memTable = processor.getWorkMemTable();
+    long memTableMapRamCost = memTable.getMemTableMapRamCost();
+    processor.insertTablet(genInsertTablePlan(1001, false), 0, 10, new TSStatus[10]);
+    Assert.assertEquals(memTableMapRamCost, memTable.getMemTableMapRamCost());
+
+    TSRecord record = new TSRecord(123L, "root.sg.device5");
+    record.addTuple(DataPoint.getDataPoint(dataType, "ns0", String.valueOf(1)));
+    processor.insert(new InsertRowPlan(record));
+    Assert.assertEquals(
+        memTableMapRamCost
+            + RamUsageEstimator.shallowSizeOfInstance(AlignedWritableMemChunk.class)
```

Comment:
```suggestion
            + RamUsageEstimator.shallowSizeOfInstance(WritableMemChunk.class)
```

And it would be better if the test case covers`calculateAlignedMemTableMapIncrement` .

Reply:
Thanks ! And this method has been renamed now.

Reply:
A new ut has been added to test ram cost of an aligned `memTableMap` now.